seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/oracle/OracleDataTypeConvertor.java

```diff
@@ -128,7 +128,6 @@ public SeaTunnelDataType<?> toSeaTunnelType(
             case ORACLE_XML:
                 return BasicType.STRING_TYPE;
             case ORACLE_DATE:
-                return LocalTimeType.LOCAL_DATE_TYPE;
```

Comment:
It is compatible for the reading data end, but may not be compatible for the downstream writing data end because the data content has changed. But the purpose of this PR is to correct previous wrong behavior.